src/components/ui/dashboard/coaching-session-list.tsx

```diff
@@ -42,8 +42,7 @@ export default function CoachingSessionList() {
     e.preventDefault();
 
     // Format the date string
-    const formattedDate = format(
-      new Date(newSessionDate),
+    const formattedDate = DateTime.fromISO(newSessionDate).toFormat(
```

Comment:
I created a nice [function to work with ISO DateTime strings here](https://github.com/refactor-group/refactor-platform-fe/blob/a0f620610517252c6d456513a7b844b55927fac8/src/types/session-title.ts#L71C5-L73C46) that I recommend using:

```typescript
const formattedDate = getDateTimeFromString(newSessionDate)
      .setLocale(locale)
      .toLocaleString(DateTime.DATETIME_SHORT);
```

More about [ts-luxon's formatting functions here](https://tonysamperi.github.io/ts-luxon/docs/#/formatting?id=tolocalestring-strings-for-humans).

Reply:
Updated! f8d2a81db08ab15150c0b075293169be70fcca3a